schemas/IDLs/org.opencb.biodata.models/1.2.0/evidence.avdl

```diff
@@ -316,12 +320,12 @@ Evidence of benign impact:
 * `SO_0002055`: null_variant (http://purl.obolibrary.org/obo/SO_0002055)
     */
     enum VariantFunctionalEffect {
-        SO_0002052,
-        SO_0002053,
-        SO_0001773,
-        SO_0002054,
-        SO_0001786,
-        SO_0002055
+        dominant_negative_variant,
+        gain_of_function_variant,
+        lethal_variant,
+        loss_of_function_variant,
+        loss_of_heterozygosity,
+        null_variant
```

Comment:
there is one of this issues which is like the Pisuerga (i.e.: appears and disappears) with the download and extraction of avdl from biodata and the quick&dirty workaround was to commit the biodata models in the repo. 
I have to fix this!